bam_qc_metrics/bam_qc.py

```diff
@@ -158,33 +188,66 @@ def cleanup(self):
         """
         if self.tmp_object != None:
             self.tmp_object.cleanup()
-        elif self.verbose:
-            sys.stderr.write("Omitting cleanup for user-specified "+\
-                             "temporary directory %s\n" % self.tmpdir)
+            self.logger.info("Cleaned up default temporary directory %s" % self.tmpdir)
+        else:
+            self.logger.info("Omitting cleanup for user-specified "+\
+                             "temporary directory %s" % self.tmpdir)
 
+    def configure_logger(self, log_path=None):
+        logger = logging.getLogger(__name__)
+        if self.debug:
+            log_level = getattr(logging, 'DEBUG')
```

Comment:
Why not `logging.DEBUG`?

Reply:
No particular reason. `logging.DEBUG` is slightly safer, will fix.